Refer to this link for build results (access rights to CI server needed): 
http://bock.systems:8080/job/hangups_tox/13/

**Build Log**
_last 25 lines_

```
[...truncated 23 lines...]
^
./hangupsbot/commands.py:524:80: E501 line too long (83 > 79 characters)
    prs = get_json('https://api.github.com/repos/TheInnerCircleO/hangupsbot/pulls')
                                                                               ^
./hangupsbot/commands.py:535:80: E501 line too long (89 > 79 characters)
        segments.append(hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK))
                                                                               ^
./hangupsbot/commands.py:536:80: E501 line too long (89 > 79 characters)
        segments.append(hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK))
                                                                               ^
./hangupsbot/commands.py:537:80: E501 line too long (81 > 79 characters)
        segments.append(hangups.ChatMessageSegment('[{}] '.format(pr['number'])))
                                                                               ^
./hangupsbot/commands.py:538:80: E501 line too long (118 > 79 characters)
        segments.append(hangups.ChatMessageSegment(pr['title'], hangups.SegmentType.LINK, link_target=pr['html_url']))
                                                                               ^
./hangupsbot/commands.py:540:80: E501 line too long (129 > 79 characters)
        segments.append(hangups.ChatMessageSegment(pr['user']['login'], hangups.SegmentType.LINK, link_target=pr['user']['url']))
                                                                               ^
ERROR: InvocationError: '/var/lib/jenkins/jobs/hangups_tox/workspace/.tox/pep8/bin/flake8'
___________________________________ summary ____________________________________
ERROR:   pep8: commands failed
Build step 'Execute shell' marked build as failure
Setting status of 2d147ef8fbebd6f703468b07238ffda2ae546c8a to FAILURE with url http://bock.systems:8080/job/hangups_tox/13/ and message: Merged build finished.
```

Oh No...

--- a/hangupsbot/commands.py
+++ b/hangupsbot/commands.py
@@ -518,10 +518,13 @@ def xkcd(bot, event, *args):
 
     bot.parse_and_send_segments(event.conv, text)
 
+
 @command.register
 def prs(bot, event, *args):
 
-    prs = get_json('https://api.github.com/repos/TheInnerCircleO/hangupsbot/pulls')
+    prs = get_json(
+        'https://api.github.com/repos/TheInnerCircleO/hangupsbot/pulls'
+    )
 
     if not prs:
         bot.parse_and_send_segments(event.conv, 'No open pull requests')
@@ -532,11 +535,34 @@ def prs(bot, event, *args):
     ]
 
     for pr in prs:
-        segments.append(hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK))
-        segments.append(hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK))
-        segments.append(hangups.ChatMessageSegment('[{}] '.format(pr['number'])))
-        segments.append(hangups.ChatMessageSegment(pr['title'], hangups.SegmentType.LINK, link_target=pr['html_url']))
+        segments.append(
+            hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK)
+        )
+
+        segments.append(
+            hangups.ChatMessageSegment('\n', hangups.SegmentType.LINE_BREAK)
+        )
+
+        segments.append(
+            hangups.ChatMessageSegment('[{}] '.format(pr['number']))
+        )
+
+        segments.append(
+            hangups.ChatMessageSegment(
+                pr['title'],
+                hangups.SegmentType.LINK,
+                link_target=pr['html_url']
+            )
+        )
+
         segments.append(hangups.ChatMessageSegment(' by '))
-        segments.append(hangups.ChatMessageSegment(pr['user']['login'], hangups.SegmentType.LINK, link_target=pr['user']['url']))
+
+        segments.append(
+            hangups.ChatMessageSegment(
+                pr['user']['login'],
+                hangups.SegmentType.LINK,
+                link_target=pr['user']['url']
+            )
+        )
 
     bot.send_message_segments(event.conv, segments)

